src/components/Operations.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { fetchOperations } from '../actions';
+import { db } from '../firebaseInit';
+
+class Operations extends Component {
+    componentDidMount() {
+        this.props.fetchOperations(db.collection('operations'));
+    }
+
+    render() {
+        return (
+            <div>
+                <h1>Operations stream</h1>
+                <span>{ JSON.stringify(this.props.ops) }</span>
```

Comment:
This will obviously change later but as a newcomer i don't understand what is worthwhile to show on each operation, since this isn't actually displaying a user-understandable thing it's not clear what this is intended to show